.upstream-tests/test/std/utilities/time/time.cal/time.cal.day/time.cal.day.members/plus_minus_equal.pass.cpp

```diff
@@ -19,6 +19,11 @@
 
 #include "test_macros.h"
 
+#if defined(_MSC_VER)
+# pragma warning( disable: 4307 )
```

Comment:
Can you add comments to these, similarly how the 4244 disables have comments?

Reply:
Added comments.